README.md

```diff
@@ -51,6 +51,19 @@ If you have previously created a Mobile Application:
 * choose Installation from the Settings section in the left nav, and
 * download the New Relic agent for your platform and follow the instructions.
 
+## Troubleshooting
+If an error such as artifact does not match checksum or cannot download framework from remoteSourceControl is encountered when adding swift package please try the following steps.
```

Comment:
If you receive an error when adding the Swift Package, such as

* `artifact does not match checksum`
* `cannot download framework from remoteSourceControl`

please try the following steps: